apps/cowswap-frontend/src/api/cowProtocol/priceApi.ts

```diff
@@ -1,46 +0,0 @@
-import { RAW_CODE_LINK } from '@cowprotocol/common-const'
-import { environmentName } from '@cowprotocol/common-utils'
-import { SupportedChainId, mapSupportedNetworks } from '@cowprotocol/cow-sdk'
-
-import { PriceStrategy as PriceStrategyEnum } from 'legacy/state/gas/atoms'
-
-const API_NAME = 'CoW Protocol'
-const STRATEGY_URL_BASE = RAW_CODE_LINK + '/configuration/config/'
-const ENV_BASE = environmentName !== 'production' ? 'barn' : environmentName
-const STRATEGY_URL = STRATEGY_URL_BASE + ENV_BASE + '/strategies'
```

Comment:
@shoom3301 Has the `configuration` branch been updated to reflect those changes?
Not removing it completely right away since it'll break previous UI versions, but for now update the readme and associated internal guides.

Well, there's no readme in that branch https://github.com/cowprotocol/cowswap/tree/configuration.

But I did find mentions to it on Notion:
- https://www.notion.so/cownation/Config-CoW-Swap-27435f03f9df4d3d8ba166aff5e2bd3e?pvs=4
- https://www.notion.so/cownation/Adapt-CoW-SDK-fe2633a150f24b678eeace2c9abab398?pvs=4

I thought there were more places where it was documented but I couldn't find it.